project_task_materials_stock/README.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+============================
+Project Task Materials Stock
+============================
+Project Tasks allow to record time spent, but some activities, such as
+Field Service, often require you to keep a record of the materials spent.
+
+This module extends project_task_material module to consume products spent in 
+a task and create analytic lines to manage costs and create invoices.  
+
+Configuration
+=============
+
+Check option 'Consume Material' in Task Stage to generate a stock move when
+the task is in that stage.
+
+Usage
+=====
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```

Comment:
Add instruction un usage to explain that there is a new field named "Consume Material" in "Task stage" to indicate when you want to consume materials